Tribler/Core/Modules/MetadataStore/store.py

```diff
@@ -436,11 +436,13 @@ def check_update_opportunity():
             if node.timestamp < payload.timestamp:
                 node.set(**payload.to_dict())
                 result.append((node, UPDATED_OUR_VERSION))
+                return result
             elif node.timestamp > payload.timestamp:
                 result.append((node, GOT_NEWER_VERSION))
+                return result
```

Comment:
I think it is relatively easy to cover this line with a unit test. Could you please do so?

Reply:
Done.